src/main/java/frc/robot/Robot.java

```diff
@@ -119,6 +131,8 @@ public void teleopInit() {
   @Override
   public void teleopPeriodic() {
     CommandScheduler.getInstance().run();
+    DrivingDeltas calculatedDeltas = vision.targetDelta();
```

Comment:
We don't need this here.  It will be done in LightsCommand, and is part of a different feature.